README.md

```diff
@@ -257,15 +267,45 @@ Building the program with BLAS support may lead to some performance improvements
     cmake --build . --config Release
     ```
 
-### Prepare Data & Run
+### Aquiring Setting up the 7b model weights
+
+You can use this system to conduct research on an AI chatbot vaguely comparable to ChatGPT-3 and it will even run on your local machine without needing massive amounts of hardware.  But to do so you **must** install the Alpaca 7b model weights into the models folder.
+
+Because these resources belong to Facebook, their official path to obtaining the data should be followed.  While it's true that most researchers using the Alpaca weights obtained them from a magnet link to a torrent file, linking or sharing that magnet link should not be done in this repo due to the questionability of violating FaceBook's IP rights and also (not to be an alarmist here) the potential for the popularization of these weights to cause harm.
+
+- **Under no circumstances should IPFS, magnet links, or any other links to model downloads be shared anywhere in this repository, including in issues, discussions, or pull requests. They will be immediately deleted.**
+- The LLaMA models are officially distributed by Facebook and will **never** be provided through this repository.
+- Refer to [Facebook's LLaMA repository](https://github.com/facebookresearch/llama/pull/73/files) if you need to request access to the model data.  Any Magnet links shared
+
+#### Putting the Model Weights in the Right Spot
+
+This guide will assume that you've downloaded the files to an arbitrary folder, `/mnt/c/ai/models/LLaMA` using some responsible means described above.
+
+Because the weight files are so large, you may find it convenient to use softlinks to make them appear within the `models/` folder instead of having to copy them around on your HDD.  Otherwise, if you prefer, you can just download the files directly in the `models/` folder and skip the below softlink command:
 
 ```bash
-# obtain the original LLaMA model weights and place them in ./models
+# On linux, run this from the root of this repo
+rm models/.gitkeep
+
+# Now that the folder is empty, this command can safely remove the models/ folder or errors if something is still there
+rm -r models/
```

Comment:
This directory contains `ggml-vocab.bin`. If we're going to recommend people delete that directory as part of "getting started" this file needs to be moved. Although maybe that's your intention.

Reply:
Why is it necessary to delete the directory, and why is it even committed to git in that case? `ggml-vocab.bin` is a model file, it contains the tokenization data.

Also, loading the models over the 9P file sharing that WSL2 uses is super slow (it basically doesn't work for mmap at all)

Reply:
It's not deleted, it was (I hope) carefully moved to `tests/`.  

I think the `ggml-vocab.bin` file was once used more generally in the preparation of data phase, but is now only used by the tests?  I didn't want to get too involved with figuring out a long term-solution about managing project-internal bin files such as including a download required bin files during testing so just moved it into `tests/` but am open to that discussion.  

Also, thanks and noted @SlyEcho I'll do some benchmarks and see if they're still having [bad performance](https://github.com/microsoft/WSL/discussions/9412) but in most cases symlinks are transparent in linux/ mac.

Reply:
@SlyEcho Holy cow, thanks for this catch about performance, I'm very grateful.  Between running WSL2 over p9 vs the native build, native is almost 10x faster.  As a linux/ mac devloper who also has a gaming machine, I liked the comfort that WSL2 provided me when onboarding, but yeah, maybe there should be a remark on 9P file sharing.